glue/core/command.py

```diff
@@ -205,9 +219,12 @@ def undo(self, session):
 class NewDataViewer(Command):
     """Add a new data viewer to the application
 
-    :param viewer: The class of viewer to create
-    :param data: The data object to initialize the viewer with, or None
-    :type date: :class:`~glue.core.data.Data` or None
+    Parameters
+    ----------
+    viewer:
```

Comment:
Here and elsewhere where the type is a viewer, the type to include in the docstring should be ``glue.viewers.common.viewer.BaseViewer``